package.json

```diff
@@ -1,59 +1,54 @@
 {
-  "name": "create-react-component-folder",
-  "version": "0.1.15",
-  "description": "Creates single react component folder structure",
+  "name": "create-react-component",
+  "version": "0.0.2",
+  "description": "Script to create a new React component",
   "main": "./lib/index",
   "readme": "README.md",
   "engines": {
-    "npm": ">=5.2.0"
+    "npm": ">=6.12.1"
   },
   "scripts": {
     "dev": "node ./lib/index",
-    "lint": "eslint lib bin"
+    "lint": "eslint lib bin",
+    "lint:fix": "eslint --fix lib bin"
   },
   "keywords": [
     "cli",
     "scaffold",
     "react",
-    "react-native",
-    "typescript",
-    "component",
-    "components",
-    "npm",
-    "crcf",
     "create-react-app",
-    "create-react-component",
-    "create-react-component-folder"
+    "create-react-component"
   ],
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/snaerth/create-react-component-folder"
+    "url": "git+https://github.com/natejcho/create-react-component.git"
   },
-  "author": "Snær Seljan Þóroddsson",
+  "author": "Nathaniel Cho",
   "license": "MIT",
   "devDependencies": {
-    "app-root-dir": "^1.0.2",
-    "eslint": "^4.17.0",
-    "eslint-config-airbnb": "^16.1.0",
-    "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-jsx-a11y": "^6.0.3",
-    "eslint-plugin-react": "^7.6.1",
-    "prettier-eslint": "^8.8.1"
+    "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.10.0",
+    "eslint-plugin-prettier": "^3.1.2",
+    "husky": "^4.0.7",
+    "lint-staged": "^10.0.0-beta.15",
+    "prettier": "^1.19.1"
   },
   "bin": {
-    "create-react-component-folder": "./bin/create-react-component-folder.js",
-    "crcf": "./bin/crcf.js"
+    "crac": "./bin/crac.js"
   },
   "dependencies": {
-    "chalk": "^2.3.0",
-    "commander": "^2.13.0",
     "fs": "0.0.1-security",
-    "log-update": "^2.3.0",
-    "mkdirp": "^0.5.1",
-    "node-alias": "^1.0.4",
-    "npmview": "0.0.4",
-    "prettier": "^1.10.2",
-    "react-dev-utils": "^5.0.0",
```

Comment:
used for clearConsole

that'd be nice but this package comes with way too many dep